docs/src/content/docs/deployment/stack-troubleshooting.md

```diff
@@ -3,7 +3,28 @@ title: Troubleshooting the stack
 description: Troubleshooting issues with the stack
 ---
 
-### Conda env not activated
+## Integration tests failing
+
+If any integration tests fail, it is usually best to go back to the first test that failed, as subesquent tests often rely on earlier tests passing to succeed. Therefore the root cause is usually with the first failure.
+
+If `test_tyk` fails, all other tests will fail because the stack relies on tyk being up and running as expected. If you see `test_tyk` fail, you may as well `ctrl+c` to stop the tests running and start investigating the issue with tyk. Some places to start looking for the issue are:
+- is the tyk container up and running?
+  - check your docker desktop or `docker ps` to see if tyk and all other containers are running
+- is there anything in the tyk docker logs that indicates an issue?
+  - view the logs with `docker logs candigv2_tyk_1` or in Docker desktop
+  - if anything looks amiss, make a github issue with the details
+- did anything go wrong in the build logs?
+  - the build log can be found in `tmp/error.txt` 
+- is anything else in the central logs looking weird?
+  - the central logs for all docker containers can be found in `/tmp/logs/`
```

Comment:
```suggestion
  - the central logs for all docker containers can be found in `/tmp/logs/buffer.***.log`
```

Reply:
I clarified this a little bit in latest commit